src/ds5/ds5-factory.cpp

```diff
@@ -634,38 +624,41 @@ namespace librealsense
 
         bool restore_color_stream_extrinsic()
         {
-            try
-            {
-                LOG_WARNING("invalid RGB extrinsic was identified, recovery routine was invoked");
+            LOG_WARNING("invalid RGB extrinsic was identified, recovery routine was invoked");
```

Comment:
Please adjust the wording ~~recovery~~ lookup
Also move it under (if(fw==5.1.6....)) condition